internal/proto/controller/api/services/v1/account_service.proto

```diff
@@ -110,10 +110,40 @@ message GetAccountResponse {
 message ListAccountsRequest {
   string auth_method_id = 1 [json_name = "auth_method_id"]; // @gotags: `class:"public"`
   string filter = 30 [json_name = "filter"]; // @gotags: `class:"sensitive"`
+  // An opaque token used to continue an existing iteration or
+  // request updated items. If paginating, use this token in the
+  // next list request.
+  string refresh_token = 40 [json_name = "refresh_token"]; // @gotags: `class:"public"`
+  // The maximum size of a page in this iteration.
+  // If unset, the default page size configured will be used.
+  // If the page_size is greater than the max page size configured,
+  // the page size will be truncated to this number.
+  uint32 page_size = 50 [json_name = "page_size"]; // @gotags: `class:"public"`
 }
 
 message ListAccountsResponse {
+  // The items returned in this page.
   repeated resources.accounts.v1.Account items = 1;
+  // The type of response, either "delta" or "complete".
+  // Delta signifies that this is part of a paginated result
+  // or an update to a previously completed pagination.
+  // Complete signifies that it is the last page.
+  string response_type = 20 [json_name = "response_type"]; // @gotags: `class:"public"`
```

Comment:
Protobuf numbering in our files is a mess in general, but stick with the convention _in the message_ if you can. This should be `2`, `3` and so on.